File: src/main/java/pw/saber/regions/utils/Logger.java
package pw.saber.regions.utils;

import org.bukkit.ChatColor;
import pw.saber.regions.InvisRegions;

/**
 * SaberInvisRegions - Developed by Driftay.
 * All rights reserved 2020.
 * Creation Date: 5/26/2020
 */
public class Logger {

    public static void print(String message, PrefixType type) {
        InvisRegions.getInstance().getServer().getConsoleSender().sendMessage(type.getPrefix() + message);
    }

    public enum PrefixType {

        WARNING(ChatColor.RED + "WARNING: "), NONE(""), DEFAULT(ChatColor.GOLD + "[InvisRegions] "), FAILED(ChatColor.RED + "FAILED: ");

        private String prefix;

        PrefixType(String prefix) {
            this.prefix = prefix;
        }

        public String getPrefix() {
            return this.prefix;
        }

    }

}

File: src/main/java/pw/saber/regions/utils/Logger.kt
package pw.saber.regions.utils

import org.bukkit.ChatColor
import pw.saber.regions.InvisRegions

/**
 * SaberInvisRegions - Developed by Driftay.
 * All rights reserved 2020.
 * Creation Date: 5/26/2020
 */
object Logger {
    @JvmStatic
    fun print(message: String, type: PrefixType) {
        InvisRegions.instance.server.consoleSender.sendMessage(type.prefix + message)
    }

    enum class PrefixType(val prefix: String) {
        WARNING(ChatColor.RED.toString() + "WARNING: "), NONE(""), DEFAULT(ChatColor.GOLD.toString() + "[InvisRegions] "), FAILED(ChatColor.RED.toString() + "FAILED: ");

    }
}